optimize/client/.eslintrc

```diff
@@ -1,14 +1,32 @@
 {
```

Comment:
I see some 'trailing comma' warning in the the eslint file for some reason. Do you face the same issue?

Reply:
yeah, I shouldn't format it so it added those commas. This file never had any trailing commas before